tracer/test/snapshots/GraphQL2Tests.SubmitsTraces.verified.txt

```diff
@@ -77,11 +79,13 @@
     Tags: {
       component: aspnet_core,
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
       http.status_code: 200,
       http.url: http://localhost:00000/shutdown,
       language: dotnet,
+      network.client.ip: 127.0.0.1,
```

Comment:
These don't have the user agent, should we add them to the graphql server spans?

Reply:
I don't know if we need to set them on spans of type:graphql, but we should on these aspnet_core.request spans

Reply:
fixed in f6d1c9dcaca4d9e3a7308c2812615047326f9f35